src/main/java/ComputerPlayer.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Random;
+
+public class ComputerPlayer implements Player {
+    private final Display display;
+    private final String mark;
+    
+    public ComputerPlayer(Display display, String mark) {
+        this.display = display;
+        this.mark = mark;
+    }
+    
+    @Override
+    public int chooseMove(Board board, String opponentMark) {
+        displayMessages(board);
+        
+        var availableSquares = board.availableSquares(mark, opponentMark);
+        Random rand = new Random();
+        var move = availableSquares.get(rand.nextInt(availableSquares.size()));
+        
+        return Integer.parseInt(move);
+    }
+    
+    @Override
+    public String getMark() {
+        return mark;
+    }
+    
+    private void displayMessages(Board board) {
```

Comment:
I like how you've separated your display logic from your logic to choose a move. What files would you need to change if you wanted to adjust how the game is displayed?

Reply:
Thanks, Ashley. I think this method would fit better within the display class. If I needed to adjust how the game is displayed, this will reduce where I need to change the code